DigitalLearningSolutions.Data/Services/PasswordResetService.cs

```diff
@@ -89,22 +89,26 @@ END CATCH
         private Email GeneratePasswordResetEmail(string emailAddress, string resetHash, string firstName, string baseUrl)
```

Comment:
Can't reply to Dan's comment, but I agree that it makes sense to change it everywhere (unless it's too much work, in which case I can flag the Frameworks changes with Kevin)